scripts/generate_v5_configuration.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+require('dotenv').config();
+const defaultConfigJson = require('../config/config.json');
```

Comment:
Should we call the Utilities.copyObject  function here?

Reply:
This will be run as separate node process so it won't affect ot-node process. So i would say that we don't need copyObject here.